src/Serilog.Sinks.AzureDataExplorer/Durable/PortableTimer.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Serilog.Debugging;
+
+namespace Serilog.Sinks.AzureDataExplorer.Durable
+{
```

Comment:
Can we import this class ? Is there a specific reason we have used this as-is in the code without any customization (excuse me if I am ignorant of something)

Reply:
Since serilog itself is a logging framework, this is a feature from serilog framework, for debugging and diagnostics where we output to selflog which needs to enabled during program startup. More on this https://github.com/serilog/serilog/wiki/Debugging-and-Diagnostics